packages/versions/src/lib/checkFuelCoreVersionCompatibility.test.ts

```diff
@@ -0,0 +1,65 @@
+import { checkFuelCoreVersionCompatibility } from './checkFuelCoreVersionCompatibility';
+import * as getSupportedVersionsMod from './getSupportedVersions';
+
+describe('getDifferenceToUserFuelCoreVersion', () => {
+  afterAll(() => jest.resetAllMocks());
```

Comment:
Apologies, I made a mistake on this one. 😓 

```suggestion
  afterAll(() => jest.restoreAllMocks());
```